### PR TITLE
proper /var/run/smt packaging

### PR DESCRIPTION
**Pull request by aaannz**

### CODE REVIEW

**Comment by jsrain** [comment]
LGTM

File: package/smt.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon Aug  3 12:17:15 UTC 2015 - oholecek@suse.com
+
+- fix /var/run/smt packaging (bsc#940343)
+  add tmpfile-smt.spec as source2
```
Comment (mcalmer) [inline package/smt.changes]: You mean tmpfile-smt.conf, right?

Comment (aaannz) [inline package/smt.changes]: Of course :) Good catch

**Comment by mcalmer** [comment]
+1 ready to merge

**Pull request merged**